cmd/scale.go

```diff
@@ -274,7 +274,7 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 
 	sc.containerService.Properties.AgentPoolProfiles = []*api.AgentPoolProfile{sc.agentPool}
 
-	template, parameters, _, err := templateGenerator.GenerateTemplate(sc.containerService, acsengine.DefaultGeneratorCode)
+	template, parameters, _, err := templateGenerator.GenerateTemplate(sc.containerService, acsengine.DefaultGeneratorCode, false)
```

Comment:
It might be interesting to check the behavior of scale / backward compatibility of scale after reverting the Azure CNI static IPs PR... Do we know that scale wasn't affected?

Reply:
Good point. My understanding is that we are only scaling agent nodes but not master nodes, so hopefully it should not be affected. I will check if we have testing coverage on this.

Reply:
Validated with scaling tests and passed.